### DISCUSSION TITLE
Question - Where to find / edit git icons. Maybe not part of this codebase?

### DISCUSSION THREAD

**Discussion by AdamBD** [kyazdani42/nvim-web-devicons#384] [Q&A] [answered]
![image](https://github.com/nvim-tree/nvim-web-devicons/assets/1134525/4abe540d-ea00-451f-b01c-509680e36dee)

Are git related icons part of this codebase? or is that handled / managed somewhere else if using nvim-tree?

**Comment by gegoune** [comment, marked as answer]
`:h nvim-tree.renderer.icons.glyphs.git`
